lib/readAccessor.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+var Cesium = require('cesium');
+var Cartesian2 = Cesium.Cartesian2;
+var Cartesian3 = Cesium.Cartesian3;
+var Cartesian4 = Cesium.Cartesian4;
+var Matrix2 = Cesium.Matrix2;
+var Matrix3 = Cesium.Matrix3;
+var Matrix4 = Cesium.Matrix4;
+var byteLengthForComponentType = require('./byteLengthForComponentType');
+var numberOfComponentsForType = require('./numberOfComponentsForType');
+var readBufferComponentType = require('./readBufferComponentType');
+
+module.exports = readAccessor;
+
+// read the accessor shown and return an array containing its contents
+// VECx types become CartesianX
+// MATx types become MatrixX
+function readAccessor(accessor, gltf) {
```

Comment:
Pass in `gltf` first.  In general, that should come first in function parameters, e.g., less specific to more specific, e.g., `(gltf, mesh, primitive)`.